publish.gradle

```diff
@@ -13,23 +13,44 @@
 // limitations under the License.
 
 apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
 afterEvaluate {
+
     publishing {
         repositories {
             maven {
+                name = 'mavenRepo'
                 url = findProperty('mavenRepo') ?: "${buildDir}/repo"
             }
+            maven {
+                def releasesRepoUrl = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+                def snapshotsRepoUrl = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+
+                url = releaseVersion.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
+
+                credentials {
+                    username = System.getenv("OSSRH_USERNAME")
+                    password = System.getenv("OSSRH_PASSWORD")
```

Comment:
ℹ️ All credentials and keys are added to the GitHub actions secrets. Later we should define a publishing flow.